source/text/WrappedText.cpp

```diff
@@ -149,6 +149,14 @@ int WrappedText::Height() const
 
 
 
+// Get the height of the wrapped text.
```

Comment:
copy/paste typo

Reply:
> Thanks! Do you have `@2x` versions of the images too?

Not yet. I'll try to get to those this evening.